python/cucim/src/cucim/skimage/_vendored/_ndimage_util.py

```diff
@@ -97,6 +103,25 @@ def _check_mode(mode):
     return mode
 
 
+def _check_axes(axes, ndim):
+    if axes is None:
+        return tuple(range(ndim))
+    elif cupy.isscalar(axes):
+        axes = (operator.index(axes),)
+    elif isinstance(axes, Iterable):
+        for ax in axes:
+            axes = tuple(operator.index(ax) for ax in axes)
+            if ax < -ndim or ax > ndim - 1:
+                raise AxisError(f"specified axis: {ax} is out of range")
+        axes = tuple(ax % ndim if ax < 0 else ax for ax in axes)
```

Comment:
Please revisit these lines if these are intended behaviors.

It is reassigning axes inside the loop while also iterating over axes so I wonder if the following statements are intended code or not.

```suggestion
        axes = tuple(operator.index(ax) for ax in axes)
        for ax in axes:
            if ax < -ndim or ax > ndim - 1:
                raise AxisError(f"specified axis: {ax} is out of range")
        axes = tuple(ax % ndim if ax < 0 else ax for ax in axes)
```